### PR TITLE
Add support for Ninja Forms 3; Close #1

### LINKED ISSUE #1: Options not showing up in field
I have three forms built but none are showing up in the selection field for Ninja Forms.

It is outputting this code.

```
<select name="acf[field_57e0760e79a8e]" id="acf[field_57e0760e79a8e]">
                    <option value="1">
                          </option>
                      <option value="2">
                          </option>
                      <option value="3">
                          </option>
                </select>
```

Here is a screenshot [http://dev.walltribe.com/wp-content/uploads/2016/09/screencapture-dev-walltribe-wp-admin-post-new-php-1474330252484.png]

Let me know if I am doing something wrong.

Thanks a ton for this plugin.

Eric

### PR DESCRIPTION
**Pull request by JacobDB**
This adds full support for Ninja Forms 3, outputting data the same way it does for Ninja Forms 2. Additionally added documentation regarding NF3 to README.

### CODE REVIEW

**Comment by JacobDB** [comment]
@bostondv would you be willing to make me a contributor on this project? I'd be happy to maintain it going forward, as it seems you've abandoned the it.

File: acf-ninja-forms-field-v4.php [hunk 1]
```diff
@@ -125,24 +125,29 @@ function create_options( $field )
...
   function create_field( $field )
   {
     /*
     *  Review the data of $field.
     *  This will show what data is available
     */
-    
+
     // vars
+    $nf_version = version_compare( get_option( 'ninja_forms_version', '0.0.0' ), '3', '<' ) || get_option( 'ninja_forms_load_deprecated', FALSE ) ? 2 : 3;
```
Comment (bostondv) [inline acf-ninja-forms-field-v4.php]: Mind adding this as a class function to DRY up this version check?

File: acf-ninja-forms-field-v4.php [hunk 2]
```diff
@@ -216,11 +223,22 @@ function format_value( $value, $post_id, $field ) {
...
+        } else {
+          $form_object = Ninja_Forms()->form( $v )->get();
+          $form = array( 'id' => $v, 'data' => $form_object->get_settings(), 'date_updated' => $form_object->get_setting( 'date_updated' ) );
+        }
+
         $value[ $k ] = $form;
       }
     } else {
-      $value = ninja_forms_get_form_by_id( $value );
+      if ($nf_Version === 2) {
```
Comment (bostondv) [inline acf-ninja-forms-field-v4.php]: Should be a lowercase "V"

**Comment by bostondv** [comment]
Thanks @JacobDB. I'm not doing a ton of WordPress work these days so yeah, have somewhat abandoned this plugin. Your contributions would be very welcome!

**Comment by JacobDB** [comment]
Fixed :)

**Pull request merged**